device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/ObjectLock.java

```diff
@@ -1,19 +0,0 @@
-// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-package com.microsoft.azure.sdk.iot.device;
-
-/** This class enables mocking of the base class wait and notify functions
-*/
-public class ObjectLock
-{
-    public void waitLock(long timeout) throws InterruptedException
-    {
-        this.wait(timeout);
-    }
-
-    public void notifyLock()
```

Comment:
The Object class in Java was basically being overriden just for the purpose of renaming wait/notify. Deleting the class since that is unnecessary